### PR TITLE
eth-probe with fixturenet

### PR DESCRIPTION
**Pull request by telackey**

### CODE REVIEW

File: app/data/container-image-list.txt [hunk 1]
```diff
@@ -14,3 +14,4 @@ cerc/fixturenet-eth-geth
 cerc/fixturenet-eth-lighthouse
 cerc/watcher-mobymask
 cerc/test-container
+cerc/eth_probe
```
Comment (dboreham) [inline app/data/container-image-list.txt:17]: Can we be consistent within the file regarding dash or underscore?

File: app/data/repository-list.txt [hunk 2]
```diff
@@ -12,3 +12,4 @@ cerc-io/laconic-cns-cli
 cerc-io/mobymask-watcher
 cerc-io/watcher-ts
 vulcanize/assemblyscript
+vulcanize/eth_probe
```
Comment (dboreham) [inline app/data/repository-list.txt:15]: Can we change the repo name to use a dash?

Comment (telackey) [reply app/data/repository-list.txt:15]: Sounds good to me, but I don't have the necessary permissions.  Whom should I ask?

File: app/data/container-image-list.txt (hunk 1 above)
Comment (telackey) [reply app/data/container-image-list.txt:17]: I can do that.  I had it matching the repo name, but it should work as a dash regardless of renaming the repo.

**Pull request merged**